Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.mm

```diff
@@ -69,5 +70,23 @@ void RemoteScrollbarsController::mouseExitedScrollbar(WebCore::Scrollbar* scroll
         scrollingCoordinator->setMouseIsOverScrollbar(scrollbar, false);
 }
 
+bool RemoteScrollbarsController::shouldScrollbarParticipateInHitTesting(WebCore::Scrollbar* scrollbar)
+{
+    // Non-overlay scrollbars should always participate in hit testing.
+    if (ScrollerStyle::recommendedScrollerStyle() != NSScrollerStyleOverlay)
```

Comment:
We should get the scrollbar style from the the `scrollbar` itself, not from `recommendedScrollerStyle()`, because this might be a custom scrollbar.

Reply:
This is checked in Scrollbar::shouldParticipateInHitTesting.

Reply:
Right, fixed.